### PR TITLE
Review Quest Lines 1000-1420: Quest Title Consistency Check 

### PR DESCRIPTION
**Pull request by ttgmichael**
Moar proofreading! Lines from Quest file to see what RTT may have missed. Verified translations with Korean text. 

This PR is mainly checking Quest Titles between files for consistency and meaning verification.

Let me know whether you see any lines that was missed.

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -3441,7 +3441,7 @@ ITEM_20150317_003440	Sapper Master's Introduction Letter
 ITEM_20150317_003441	{memo X}Letter of introduction that Sapper Master wrote to the Highlander Master 
 ITEM_20150317_003442	Highlander Master's Proof 
 ITEM_20150317_003443	{memo X}Confirmation from the Highlander Master that you accomplished all of the necessary requirements for advancement. 
-ITEM_20150317_003444	Tanu's Purifying Stone
+ITEM_20150317_003444	Tanu's Purifying Stone
```
Comment (Sunaries) [inline ITEM.tsv]: Extra space?

File: ITEM.tsv [hunk 2]
```diff
@@ -3562,13 +3562,13 @@ ITEM_20150317_003561	Shabby-Looking Necklace
 ITEM_20150317_003562	{memo X}On the back "To ...oved Rukl..s, Lydia." is visible. 
 ITEM_20150317_003563	Small Demon's Skirt Hem
 ITEM_20150317_003564	{memo X}A skirt hem obtained by defeating Pawndel and Pawnd. An ingredient needed to make the Demon Transform Scroll. 
-ITEM_20150317_003565	Pawnd's Soul
+ITEM_20150317_003565	Pawnd's Soul
```
Comment (Sunaries) [inline ITEM.tsv]: Extra space?

File: ITEM.tsv [hunk 3]
```diff
@@ -3562,13 +3562,13 @@ ITEM_20150317_003561	Shabby-Looking Necklace
 ITEM_20150317_003562	{memo X}On the back "To ...oved Rukl..s, Lydia." is visible. 
 ITEM_20150317_003563	Small Demon's Skirt Hem
 ITEM_20150317_003564	{memo X}A skirt hem obtained by defeating Pawndel and Pawnd. An ingredient needed to make the Demon Transform Scroll. 
-ITEM_20150317_003565	Pawnd's Soul
+ITEM_20150317_003565	Pawnd's Soul 
 ITEM_20150317_003566	A demon's soul collected in order to identify demon monsters.
 ITEM_20150317_003567	{memo X}Soul of Pawndel  
-ITEM_20150317_003568	Corylus' Soul
+ITEM_20150317_003568	Corylus' Soul
```
Comment (Sunaries) [inline ITEM.tsv:3568]: Extra space?

File: ITEM.tsv [hunk 4]
```diff
@@ -3581,7 +3581,7 @@ ITEM_20150317_003580	{memo X}Wooden arrow used by High Vubbe Archers. It might b
 ITEM_20150317_003581	{memo X}Ingredient necessary to make the Crude Divining Rod used to search for the First Tombstone Fragment. You need to collect a lot of them. 
 ITEM_20150317_003582	Crude Divining Rod 
 ITEM_20150317_003583	{memo X}A divining rod used to search for the First Tombstone Fragment which has been hidden somewhere underground. Looks weak enough to be broken by even the slightest amount of force. 
-ITEM_20150317_003584	Dusty Fragment
+ITEM_20150317_003584	Dusty Fragment
```
Comment (Sunaries) [inline ITEM.tsv]: Extra space?

File: ITEM.tsv [hunk 5]
```diff
@@ -3595,7 +3595,7 @@ ITEM_20150317_003594	Carbonized Tombstone Fragment
 ITEM_20150317_003595	{memo X}The third part of the lost Ruklys monument 
 ITEM_20150317_003596	Fourth Tombstone Fragment 
 ITEM_20150317_003597	{memo X}The fourth part of the lost Ruklys monument 
-ITEM_20150317_003598	Ruins Fragment
+ITEM_20150317_003598	Ruins Fragment
```
Comment (Sunaries) [inline ITEM.tsv]: Extra space?

File: ITEM.tsv (hunk 1 above)
Comment (ttgmichael) [inline ITEM.tsv]: I was pretty sure item names were suppose to have an extra space after their name? Most of them do.

Comment (Sunaries) [inline ITEM.tsv]: I've encountered many unnecessary spaces in item names. I've removed many of the extra spaces in my past PRs.

e.g. Tanu's Purifying Stone   (0/1)  (_I guess Github doesn't accept extra spaces.._)

compared to

Tanu's Purifying Stone (0/1)

But you probably know what I mean.

Comment (ttgmichael) [inline ITEM.tsv]: Ok, I guess this should be done with RTT :p Reverting

**Comment by Sunaries** [comment]
:+1: Good to merge.

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**